raiden_contracts/tests/fixtures/contracts.py

```diff
@@ -0,0 +1,106 @@
+import pytest
+import logging
+from web3.utils.events import get_event_data
+from eth_utils import is_address
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture
+def contracts_manager():
+    """Reimplement this - fixture should return raiden-lib's contract manager.
+        (return ContractManager(CONTRACTS_SOURCE_DIRS))"""
+    raise NotImplementedError(
+        'In order to use contract fixtures please override contracts_manager fixture.'
+    )
+
+
+@pytest.fixture
+def contract_deployer_address(faucet_address) -> str:
```

Comment:
We have this defined in https://github.com/raiden-network/raiden-contracts/blob/a17d6d908fa85d9cf963f81e8fc0e7a76aaa33fb/raiden_contracts/tests/conftest.py#L8, should be moved here.

Reply:
this is a project-specific variable, each project should redefine this fixture (I'll probably move it to libs)